app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb

```diff
@@ -0,0 +1,279 @@
+module Hyrax
+  module Dashboard
+    module CollectionsControllerBehavior
```

Comment:
You don't need an example because `#prepend` is not what Hyrax would use, it is what a consumer would use to get the effect they previously might need the separate Behavior module for.  Our stack has been counterproductively indirect with Behavior modules that are used only once and still tightly bound to the context of the consuming class.  That is to say, they are not modular modules.  Basically, all you need to do is move the guts back into the proper Controller.